ghost/activitypub/test/ActivityPubService.test.ts

```diff
@@ -0,0 +1,289 @@
+import assert from 'assert/strict';
+import {ActivityPubService} from '../src';
+import knex, {Knex} from 'knex';
+import {IdentityTokenService} from '@tryghost/identity-token-service';
+import nock from 'nock';
+
+async function getKnexInstance() {
+    const knexInstance = knex({
+        client: 'sqlite',
+        connection: {
+            filename: ':memory:'
+        },
+        useNullAsDefault: true
+    });
+
+    await knexInstance.schema.createTable('users', (table) => {
+        table.string('id').primary();
+        table.string('email');
+    });
+
+    await knexInstance.schema.createTable('integrations', (table) => {
+        table.string('id').primary();
+        table.string('slug');
+        table.string('type');
+    });
+
+    await knexInstance.schema.createTable('webhooks', (table) => {
+        table.string('id').primary();
+        table.string('event');
+        table.string('target_url');
+        table.string('api_version');
+        table.string('name');
+        table.string('secret');
+        table.string('integration_id');
+        table.datetime('created_at');
+        table.string('created_by');
+    });
+
+    return knexInstance;
+}
+
+async function addOwnerUser(knexInstance: Knex) {
+    await knexInstance.insert({
+        id: '1',
+        email: 'owner@user.com'
+    }).into('users');
+}
+async function addActivityPubIntegration(knexInstance: Knex) {
+    await knexInstance.insert({
+        id: 'integration_id',
+        slug: 'ghost-activitypub',
+        type: 'internal'
+    }).into('integrations');
+}
+
+describe('ActivityPubService', function () {
+    it('Can initialise the webhooks', async function () {
+        const knexInstance = await getKnexInstance();
+        await addOwnerUser(knexInstance);
+        await addActivityPubIntegration(knexInstance);
+
+        const siteUrl = new URL('http://fake-site-url');
+        const scope = nock(siteUrl)
```

Comment:
Out of curiosity, why called `scope`?

Reply:
That's the name that `nock` uses in their README